notebooks/PARSE-KFF-DATA.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 105,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bs4 import BeautifulSoup\n",
+    "import requests\n",
+    "import re\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Functions"
+   ]
+  },
```

Comment:
we need a section for **papermill** to define the output folder. see other notebooks for example. You should set `parameters` tag on that cell